src/HttpReply.h

```diff
@@ -132,6 +132,10 @@ class HttpReply: public Http::Message
     /// \retval 0 implies that more data is needed to parse the response prefix
     size_t parseTerminatedPrefix(const char *, size_t);
 
+    /// approximate size of a "status-line CRLF headers CRLF" sequence
+    /// \sa HttpRequest::prefixLen()
+    size_t prefixLen() const;
```

Comment:
Regarding "approximate": It would be nice to return the exact packInto() length here, but the existing code is not ready for that: This method uses a combination of the existing StatusLine::reason() and Http::One::ResponseParser::firstLineSize() code. That combination does not quite match the existing packInto() code. In very rare cases, packed prefix length will be different. This approximation is _not_ a problem for the existing method callers, and fixing this mismatch would require non-trivial and controversial changes.

Reply:
Case in point what I am requesting you do **not** try to (de-)duplicate the parser input normalization calculation for this on-wire output calculation.